notebooks/csp/arc_consistency.ipynb

```diff
@@ -39,10 +39,9 @@
   },
```

Comment:
I think the only change to arc_consistency.ipynb should be adding the line `"con_solver.visualizer.text_size = 12\n",`. The other stuff shouldn't be committed.

Reply:
done